code/game/objects/items/teleportation.dm

```diff
@@ -317,7 +317,7 @@
 			get_fragged(user, destination) //EMP teleported you into a wall? Wearing a BoH? You're dead.
 	else
 		telefrag(destination, user)
-		do_teleport(C, destination, channel = TELEPORT_CHANNEL_FREE)
+		do_teleport(C, destination, channel = TELEPORT_CHANNEL_BLINK)
```

Comment:
Thought we were getting a way to counter the syndicate tele, but I guess not

Reply:
It does counter the syndicate tele, it it doesn't block FREE or WORMHOLE teleportations. (Originally it was bluespace, but I included the rest apart from the ones that shouldn't be)